Gems/GeoJSONSpawner/Code/Source/GeoJSONSpawner/GeoJSONSpawnerComponent.h

```diff
@@ -98,5 +98,14 @@ namespace GeoJSONSpawner
 
         // Terrain notify
         bool m_terrainCreatedOnlyOnce{ false }; //!< Is terrain fully generated once
+
+        // Spawn & Despawn notify
+        GeoJSONUtils::SpawnDespawnStatus m_spawnStatus = GeoJSONUtils::SpawnDespawnStatus::Success;
```

Comment:
```suggestion
        GeoJSONUtils::SpawnDespawnStatus m_spawnStatus{ GeoJSONUtils::SpawnDespawnStatus::Success };
```
Just to follow a convention.